<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cart</class>
 <widget class="QWidget" name="Cart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:white;</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>381</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="welcomeLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>60</y>
     <width>161</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnRemoveSelected">
      <property name="styleSheet">
       <string notr="true">QPushButton{
font: 63 14pt &quot;georgia&quot;;
color: white;
padding : 0 8px;

}
QPushButton:hover:!pressed
{
background:#FFD700;
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnBorrowAll">
      <property name="styleSheet">
       <string notr="true">QPushButton{
font: 63 14pt &quot;georgia&quot;;
color: white;
padding : 0 8px;

}
QPushButton:hover:!pressed
{
background:#FFD700;
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>Borrow all</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="borrowButton">
      <property name="styleSheet">
       <string notr="true">QPushButton{
font: 63 14pt &quot;georgia&quot;;
color: white;
padding : 0 8px;

}
QPushButton:hover:!pressed
{
background:#FFD700;
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>Borrow </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnClearCart">
      <property name="styleSheet">
       <string notr="true">QPushButton{
font: 63 14pt &quot;georgia&quot;;
color: white;
padding : 0 8px;

}
QPushButton:hover:!pressed
{
background:#FFD700;
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>Vider</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="voirMonPanierButton">
      <property name="styleSheet">
       <string notr="true">QPushButton{
font: 63 14pt &quot;georgia&quot;;
color: white;
padding : 0 8px;

}
QPushButton:hover:!pressed
{
background:#FFD700;
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>cart</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnRetour">
      <property name="styleSheet">
       <string notr="true">QPushButton{
font: 63 14pt &quot;georgia&quot;;
color: white;
padding : 0 8px;

}
QPushButton:hover:!pressed
{
background:#FFD700;
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>&lt;----</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
